src/Tribe/Attendee_Registration/Modal.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Attendee Registration Modal class
+ *
+ * @since TBD
+ */
+class Tribe__Tickets__Attendee_Registration__Modal {
```

Comment:
Should we be using a namespace now?

Reply:
I looked to doing it, but then it would be the only one in the AR service provider that way. This is working now and I could see doing namespace for all those files together.